packages/MainUI/components/Toolbar.tsx

```diff
@@ -34,6 +35,13 @@ const iconMap: Record<IconName, React.FC<unknown>> = {
 
 export const Toolbar: React.FC<ToolbarProps> = ({ windowId, tabId }) => {
   const { toolbar, loading } = useToolbar(windowId, tabId);
+  const router = useRouter();
+
+  const handleAction = (action: string) => {
+    if (action === 'NEW') {
```

Comment:
could you use enums instad of hardcoding the `NEW` string here?

Reply:
I think is better to use the constants that I've created, but forgot to update this, tanks for pointing it